index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <div class="container">
+        <div><img class="icon" src="geometry.png" alt=""></div>
```

Comment:
Kindly, I suggest you use a HEADER tag, and within it a NAV tag to contain the "hamburger" menu

Reply:
Kindly, I suggest you check this image link because the image does not exist

Reply:
Good job following the advice of the previous reviewer